docs/github_actions.md

```diff
@@ -5,6 +5,6 @@
 2. The action is triggered whenever a commit is made to the `master` branch.  E.g., when a pull request is merged to `master`.
 3. OpenShift projects can periodically pull this image from GHCR.  Configure only **_NON-PRODUCTION_** MyLA projects to pull the image…
     ```sh
-    oc tag ghcr.io/tl-its-umich-edu/my-learning-analytics:latest my-learning-analytics:latest --scheduled
+    oc tag ghcr.io/tl-its-umich-edu/my-learning-analytics:latest my-learning-analytics:latest --scheduled --reference-policy=local
```

Comment:
Could you please put a comment here as for why "--reference-policy=local" is added?

Reply:
Done. https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1554/commits/ca01926cd8de9537ed3c7edf31d2288e20e641f6